quarkchain/evm/state.py

```diff
@@ -248,6 +258,7 @@ def __init__(
         qkc_config=None,
         executing_on_head=False,
         db=None,
+        is_testing=False,
```

Comment:
Using the name is_testing may be confusing.  I would suggest naming it to "use_conv_evm_account" or something like that.